### PR TITLE
[improvement] enable visualize api from seperate server

### PR DESCRIPTION
**Pull request by atb00ker**

### CODE REVIEW

**Comment by atb00ker** [comment]
Tested this with `docker-openwisp` and `topology/` visualization views are working! :smile:

File: django_netjsongraph/visualizer/generics.py [hunk 1]
```diff
@@ -19,7 +19,11 @@ def get(self, request, pk):
         topology = get_object_or_404(self.topology_model, pk)
         return render(request, 'netjsongraph/detail.html', {
             'topology': topology,
-            'graph_url': reverse('network_graph', args=[topology.pk]),
-            'history_url': reverse('network_graph_history', args=[topology.pk]),
+            'graph_url': reverse('network_graph',
+                                 urlconf=TOPOLOGY_API_URLCONF,
+                                 args=[topology.pk]),
+            'history_url': reverse('network_graph_history',
```
Comment (nemesisdesign) [inline django_netjsongraph/visualizer/generics.py]: what about the base URL? is this enough to make the application work also if the API has a different host?
Would this `reverse` call generate a path which would imply requests to the same host?

**Comment by atb00ker** [comment]
> Looks good, have you tested it also on openwisp-network-topology?

Yes, just done testing with both docker-openwisp and openwisp-network-topology.

**Comment by nemesisdesign** [comment]
I forgot this one, please remind me of other pending PRs.

**Pull request merged**